---
title: System.IdentityModel.IdentityModelAsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: e4fc924f-3b9e-44f5-97e4-b3a4bb855d89
ms.openlocfilehash: 92ef43c0105b8c9ca54c49ed70fd77c3a1ac41ed
ms.sourcegitcommit: 5b6d778ebb269ee6684fb57ad69a8c28b06235b9
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 04/08/2019
ms.locfileid: "59119175"
---
# <a name="systemidentitymodelidentitymodelasynccallbackthrewexception"></a>System.IdentityModel.IdentityModelAsyncCallbackThrewException
System.IdentityModel.IdentityModelAsyncCallbackThrewException  
  
## <a name="description"></a>Описание  
 Асинхронный обратный вызов сгенерировал исключение.  
  
## <a name="see-also"></a>См. также

- [Трассировка](../../../../../docs/framework/wcf/diagnostics/tracing/index.md)
- [Использование трассировки для устранения неполадок приложения](../../../../../docs/framework/wcf/diagnostics/tracing/using-tracing-to-troubleshoot-your-application.md)
- [Администрирование и диагностика](../../../../../docs/framework/wcf/diagnostics/index.md)
